packages/manager/apps/hpc-vmware-managed-vcd/src/components/tiles/organization-data-tile/OrganizationDataProtectionTile.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { render } from '@testing-library/react';
+import { describe, expect, it } from 'vitest';
+import {
+  ODS_THEME_TYPOGRAPHY_LEVEL,
+  ODS_THEME_TYPOGRAPHY_SIZE,
+} from '@ovhcloud/ods-common-theming';
+import React from 'react';
+import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import OrganizationDataProtectionTile from './OrganizationDataProtectionTile.component';
+
+const renderComponent = () => {
+  const queryClient = new QueryClient();
+
+  return render(
+    <QueryClientProvider client={queryClient}>
+      <OrganizationDataProtectionTile />
+    </QueryClientProvider>,
+  );
+};
+
```

Comment:
you should add also a test to check  the display regarding `useManagedVcdOrganizationBackup` data value